### DISCUSSION TITLE
更新teams

### DISCUSSION THREAD

**Discussion by lovelyCARDINAL** [MoegirlPediaInterfaceAdmins/MoegirlPediaInterfaceCodes#355] [General]
如题，更新[teams](https://github.com/orgs/MoegirlPediaInterfaceAdmins/teams)，看上去所有组都需要更新 @AnnAngela

**Comment by lovelyCARDINAL** [comment]
要不给我个org maintainer吧（

**Reply by AnnAngela** [reply]
列个列表吧2333

**Reply by lovelyCARDINAL** [reply]
ia - 乐然，bb2
patroller + 乐然，bb2
sysop - 乐然
tech + 乐然，bb2
